packages/ia-components/sandbox/AnchorDetails.jsx

```diff
@@ -15,29 +35,32 @@ export default class AnchorDetails extends IAReactComponent {
     */
     constructor(props)
     {
-        super(props);
+        super(props); // { identifier, reload }
+        this.setState({
+            urlProps: ObjectFilter(this.props, (k,v) => AnchorDetails.urlparms.includes(k)),
```

Comment:
`urlparms` could be renamed `urlparams` to be more understandable.

Reply:
Rejected: non functional suggestion and personal opinion on how to abbreviate "parameters"